addons/website_event_sale/views/website_sale_templates.xml

```diff
@@ -59,22 +59,24 @@
                             <t t-set="use_text_align" t-value="True"/>
                             <t t-set="additionnal_classes" t-value="'o_wevent_event_main_cover o_wevent_event_main_cover_top overflow-hidden h-auto'"/>
 
-                            <div t-attf-class="o_wevent_event_date position-absolute shadow-sm o_not_editable">
-                                <span t-out="event.date_begin" t-options="{'widget': 'datetime', 'tz_name': event.date_tz, 'format': 'LLL'}" class="o_wevent_event_month"/>
-                                <span t-out="event.date_begin" t-options="{'widget': 'datetime', 'tz_name': event.date_tz, 'format': 'dd'}" class="o_wevent_event_day oe_hide_on_date_edit"/>
-                            </div>
+                            <a class="text-decoration-none" t-att-href="event.website_url">
```

Comment:
Are there situations (with all modules installed) or incoming work that adds buttons, other <a> tags or inputs inside this div? If so, having a <a> tag is an issue. Otherwise, we could go with it. It is just the top / image part right? then it should be fine

Reply:
It's just the header part of the card (the cover) so it should be fine !